app/views/users/index.html.erb

```diff
@@ -1 +1,9 @@
-<h1>List of all users in the database</h1>
\ No newline at end of file
+<div class="main_container">
+<h1>My Blog App</h1>
+
+<div class = "users_list">
+<%= @users.each do |user| %>
```

Comment:
- [ ] There is an issue with the line <%= @user.each do |user| %>. The `each` method returns the original array and not nil, so it will be rendered in the view. To fix this, you can change <%= to <% to prevent the array from being rendered. 👇🏽 

![image](https://github.com/edubew/blog-box/assets/58642949/24982aa9-0986-4eef-b6ab-cf7c90cb5e01)